src/wp-includes/block-editor.php

```diff
@@ -192,12 +192,17 @@ function get_default_block_editor_settings() {
 	// These styles are used if the "no theme styles" options is triggered or on
 	// themes without their own editor styles.
 	$default_editor_styles_file = ABSPATH . WPINC . '/css/dist/block-editor/default-editor-styles.css';
-	if ( file_exists( $default_editor_styles_file ) ) {
+
+	static $default_editor_styles_file_contents = false;
```

Comment:
Nitpick: I would have set this to `''` (an empty string) for the default value, so the type of the variable doesn't change.